conftest.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import os
+from typing import Any
+
+import pytest
+import requests
+
+import openeo
+from openeo.capabilities import ComparableVersion
+
+_log = logging.getLogger(__name__)
+
+os.environ["ENDPOINT"] = "https://openeo.dataspace.copernicus.eu/"
+
+def get_openeo_base_url(version: str = "1.1.0"):
+    try:
+        endpoint = os.environ["ENDPOINT"].rstrip("/")
```

Comment:
can you use `OPENEO_BACKEND_URL` as env var for clarity instead of `ENDPOINT` (which is overly generic)